include/arch/arm/cortex_m/mpu/nxp_mpu.h

```diff
@@ -26,8 +26,16 @@
 #define MPU_REGION_SU    ((3 << 3) | (3 << 9) | (3 << 15) | (3 << 21))
 
 /* Some helper defines for common regions */
+#if defined(CONFIG_MPU_ALLOW_FLASH_WRITE)
+/* Required to initialize the flash driver */
+#define REGION_RAM_ATTR (MPU_REGION_READ | MPU_REGION_WRITE | \
+			 MPU_REGION_EXEC |  MPU_REGION_SU)
```

Comment:
I agree with Maureen.  I'd think about a more descriptive config name if that is easy to deal with.  Otherwise just deal with it in the commit msg.